server/controllers/chartsDataController/chartsDataController.test.js

```diff
@@ -76,21 +75,25 @@ describe('chartsDataController', () => {
             query: {
               filters: {
                 chrcrit_part: [
-                  { name: 'HC', value: FALSE_STRING },
-                  { name: 'CHR', value: FALSE_STRING },
-                  { name: 'Missing', value: FALSE_STRING },
+                  { label: 'HC', value: '0' },
```

Comment:
Why `0` and `1` rather than booleans? It's all JSON/JS/Mongo, if we're changing the nature of the data we can use a boolean data type right?

Reply:
The qs library stringfies everything... and using 0 and 1 works better than a true string